build.sbt

```diff
@@ -76,7 +76,7 @@ lazy val actor = akkaModule("akka-actor")
     }
   )
   .settings(VersionGenerator.settings)
-  .enablePlugins(BoilerplatePlugin)
+  .enablePlugins(BoilerplatePlugin, AkkaDisciplinePlugin)
```

Comment:
This is not needed, as the plugin is automatically enabled.

Reply:
Thanks, that was a holdover from an earlier change :)